Sming/SmingCore/Platform/Station.cpp

```diff
@@ -70,7 +70,10 @@ bool StationClass::config(String ssid, String password, bool autoConnectOnStartu
 		memset(config.password, 0, sizeof(config.password));
 		config.bssid_set = false;
 		strcpy((char*)config.ssid, ssid.c_str());
-		strcpy((char*)config.password, password.c_str());
+		if (password.length() == 64)
+			memcpy((char*)config.password, password.c_str(), 64);
```

Comment:
The issue with this is that station_config::password is exactly 64 chars so you will have no ending zero... IMHO the correct fix would be to declare uint8 password[__65__]; in esp_sta.h, and signal a bug to espressif.